### PR TITLE
Updated the ReadMe Documentation

### PR DESCRIPTION
**Pull request by davidnjagah**
Summary
Updated the documentation to reflect the update of the app from pcdc to gigameter.

Related Issue

Proposed Changes

Screenshots (if applicable)

Checklist
 I have tested these changes locally.
 I have updated the documentation (if applicable).
 I have reviewed my code changes.

Additional Notes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,148 +18,194 @@
...
-and create the windows build (.exe or msi) by runing the following command:
+- After successfully installing the app, it will send schools' internet connectivity information to Project Connect twice a day. 
 
-```
-npm run electron:make
-```
+#### Development Commands 
 
-The windows build can be found inside `/electron/dist/folder`
+<b>Basic development commands</b>
```
Comment (Victor1Ja) [inline README.md:124]: Warp code commands inside code block
ex:
```
npm install
```

Comment (davidnjagah) [reply README.md:124]: This is well noted, how should i markup the code commands without bringing conflicts? thanks

File: README.md [hunk 2]
```diff
@@ -16,148 +18,194 @@
...
+#### Development Commands 
 
-The windows build can be found inside `/electron/dist/folder`
+<b>Basic development commands</b> 
 
-## Features
-* Checks school internet connection using minimal data (obtains metrics such as upload and download speed, latency, IP address, etc)
-* Automatically detects the country of a user 
-* Uploads results to a public database
+- After cloning the code from the GitHub repository https://github.com/unicef/project-connect-daily-check-app, install all the dependencies modules by running this command: npm install
```
Comment (Victor1Ja) [inline README.md:126]: ex:

- After cloning the code from the GitHub repository https://github.com/unicef/project-connect-daily-check-app, install all the dependencies modules by running this command: 
```
npm install 
```

**Comment by davidnjagah** [comment]
Hi @Victor1Ja i've made the changes please confirm if it's okay for merging, thanks

File: README.md (hunk 2 above)
Comment (davidnjagah) [reply README.md]: Noted, please check the edits and tell me if all is in par

**Pull request merged**